app/merginapi.cpp

```diff
@@ -420,6 +419,24 @@ void MerginApi::uploadProject( const QString &projectNamespace, const QString &p
   }
 }
 
+
+bool MerginApi::infoProject( const QString &projectFullName )
```

Comment:
This is a bit confusing - to have infoProject() and signal infoProjectFinished() .... this operation is not asynchronous, so why not directly return the result? And do we even need this function? There is already MerginApi::getLocalProject() and MerginApi::localProjectChanges() - so why not call those directly from the project status model? The model could have an invokable function that would be called from QML and would populate the model contents.